src/test/java/uk/gov/hmcts/ccd/domain/service/callbacks/CallbackInvokerWireMockTest.java

```diff
@@ -95,15 +95,10 @@ public void shouldNotRetryWhenCallbackRetriesDisabled() throws Exception {
 
         List<Integer> disabledRetries = Lists.newArrayList(0);
         caseEventDefinition.setRetriesTimeoutAboutToStartEvent(disabledRetries);
-        Instant start = Instant.now();
 
         CallbackException callbackException = assertThrows(CallbackException.class, () ->
             callbackInvoker.invokeAboutToStartCallback(caseEventDefinition, caseTypeDefinition, caseDetails, false));
         assertEquals("Callback to service has been unsuccessful for event Test", callbackException.getMessage());
-
-        final Duration between = Duration.between(start, Instant.now());
```

Comment:
Can you please validate whether we need the assertion on duration in shouldRetryOnErrorWithIgnoreWarningFalseAndDefaultRetryContext test?